content/docs/tutorials/getting-started-with-cert-manager-on-azure-kubernetes-service-using-lets-encrypt-for-ssl/README.md

```diff
@@ -0,0 +1,735 @@
+    --role "DNS Zone Contributor" \
+    --assignee $USER_ASSIGNED_IDENTITY_CLIENT_ID \
+    --scope $(az network dns zone show --name $DOMAIN_NAME -o tsv --query id)
+```
+
+> 📖 Read [What are managed identities for Azure resources?](https://learn.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview)
+> for an overview of managed identities and their uses.
+>
+> 📖 Read [Azure built-in roles](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles) to learn about the "DNS Zone Contributor" role.
+
+## Add a federated identity
+
+Now we will configure Azure to trust certain Kubernetes ServiceAccount tokens,
+in particular, the service account tokens from our specific Kubernetes cluster,
+and only tokens which are associated with the cert-manager ServiceAccount.
+cert-manager will authenticate to Azure using an short lived Kubernetes ServiceAccount token,
+and it will be able to impersonate the managed identity that you created in the previous step.
+
+First export the following environment variables containing the name and namespace of the Kubernetes ServiceAccount used by the cert-manager controller:
+
+```bash
+export SERVICE_ACCOUNT_NAME=cert-manager # ℹ️ This is the default Kubernetes ServiceAccount used by the cert-manager controller.
+export SERVICE_ACCOUNT_NAMESPACE=cert-manager # ℹ️ This is the default namespace for cert-manager.
+```
+
+Then configure the managed identity to trust the cert-manager Kubernetes ServiceAccount,
+by supplying its "subject" (the distinguishing name of the Kubernetes ServiceAccount)
+and its "issuer" (a URL at which the JWT signing certificate and other metadata can be downloaded):
+
+```bash
+export SERVICE_ACCOUNT_ISSUER=$(az aks show --resource-group $AZURE_DEFAULTS_GROUP --name $CLUSTER --query "oidcIssuerProfile.issuerUrl" -o tsv)
+az identity federated-credential create \
+  --name "cert-manager" \
+  --identity-name "${USER_ASSIGNED_IDENTITY_NAME}" \
+  --issuer "${SERVICE_ACCOUNT_ISSUER}" \
+  --subject "system:serviceaccount:${SERVICE_ACCOUNT_NAMESPACE}:${SERVICE_ACCOUNT_NAME}"
+```
+
+> 📖 Read about [Workload identity federation](https://learn.microsoft.com/en-us/azure/active-directory/develop/workload-identity-federation) in the Microsoft identity platform documentation.
+
+## Reconfigure cert-manager
+
+We will label the cert-manager controller ServiceAccount for the attention of the Azure Workload Identity webhook,
+which will result in the cert-manager controller Pod having an extra volume containing a Kubernetes ServiceAccount token which it will use to authenticate with Azure.
+
+```
+helm upgrade cert-manager jetstack/cert-manager \
+    --install \
+    --create-namespace \
+    --wait \
+    --namespace cert-manager \
+    --set installCRDs=true \
+    --set-string 'serviceAccount.labels.azure\.workload\.identity/use=true' # ❗ Label the cert-manager service account for the attention of workload-identity webhook
```

Comment:
There's upcoming change in workload identity webhook, which would require the label to rather be set on pods (https://github.com/Azure/azure-workload-identity/issues/601). For the transition period, any of those would work (serviceAccount or pod label: https://github.com/Azure/azure-workload-identity/blob/df6362a239c5d3fc19f48ff8bc7ae2ced594f116/pkg/webhook/webhook.go#L136). Since we never know which version of the webhook is installed in a cluster, I guess we can make the example universal by simply setting the label in two places. 
```suggestion
    --set-string 'podLabels.azure\.workload\.identity/use=true' \
    --set-string 'serviceAccount.labels.azure\.workload\.identity/use=true' # ❗ Label the cert-manager service account and pod for the attention of workload-identity webhook
```

Reply:
Done. I've moved these to a separate values.yaml file for easier readability.